packages/create-permissionless-app/template/extras/bundler/pimlico/config.ts

```diff
@@ -0,0 +1,8 @@
+import { createPimlicoBundlerClient } from "permissionless/clients/pimlico"
+import { http } from "viem"
+import { goerli } from "viem/chains"
+
+export const bundlerClient = createPimlicoBundlerClient({
+    chain: goerli,
```

Comment:
wagmi config is with sepolia, we might want to change this to sepolia too.

Also we can use transport as:

`https://api.pimlico.io/v1/sepolia/rpc?apikey=API_KEY`